features/digital_summary.feature

```diff
@@ -12,3 +12,13 @@ Feature: Digital Appointment Summaries
       | requested_digital |
       | requested         |
       | did not request   |
+
+  Scenario: Email notification of summary document location
+    Given the customer requested a digital appointment summary
+    When I create their record of guidance
+    Then the customer should be notified
+
+  Scenario: No email notification when postal notification requested
+    Given the customer did not requested a digital appointment summary
+    When I create their record of guidance
+    Then the customer should not be notified
```

Comment:
Having just spoken in person about this it seems the best approach is to rename this entire feature to 'Delivery methods' which then positively tests the behaviour of each delivery method (digital/postal).

For testing the inverse of these (i.e. what happens if NOT digital), then this should probably be done at a unit test level.